ERCS/erc-7007.md

```diff
@@ -133,14 +170,42 @@ ERC-7007 Metadata JSON Schema for reference
         "aigc_data": {
             "type": "string",
             "description": "A URI pointing to a resource with mime type image/* representing the asset to which this AIGC NFT represents."
+        },
+        "proof_type": {
+            "type": "string",
+            "description": "validity (zkML) or fraud (opML)"
         }
     }
 }
 ```
 
 ## Rationale
 
-TBD
+There are many proposed uses of non-fungible tokens (NFT) that depend on AI-generated content (AIGC). While traditional AIGC-based NFT can be easily implemented under the [ERC-721](./eip-721.md) schema, NFT holders don’t have the right to choose the prompt, which is the most creative part, given a pre-defined Machine Learning model. We want to give the right of prompt creation back to the NFT holders who don't have to afford the inference cost. Therefore we propose this verifiable AIGC-NFT interface to help separate prompt from the ML inference. The ecosystem will be stronger if we have a standardized interface that allows prompt-bound NFT.
```

Comment:
This chunk here definitely belongs in Motivation, because it is justifying the proposal as a whole. The stuff in your Rationale section should explain technical choices made within the EIP itself.